legacy/make_conserved_intron_gtf.py

```diff
@@ -27,7 +27,7 @@
 
 	count+=1
 	if count%10000:
```

Comment:
`== 0:`

Reply:
I saw this, but deliberately decided to skip the fix... Will remove this in next PR anyway.